assignments/objects.js

```diff
@@ -19,27 +19,79 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
```

Comment:
Key value pairs can be defined as ```id: 1```.  You only need the keys in quotations if you are needing to pass objects as a JSON object.